### ISSUE TITLE
New conversation from mobile app

### ISSUE THREAD

**Issue by fmbislandit** [chatwoot/chatwoot-mobile-app#637]
Feature request to be able to start a new conversation and send an outbound message from the mobile app. This can currently be accomplished in the web browser under a contact but mobile app only has the ability to reply to a message, not create a new one

**Comment by velatechs** [comment]
Its really a useful features.

**Comment by yuralves** [comment]
Is somebody currently working on this feature?

**Comment by muhsin-k** [comment]
Yes. This is one of the most requested features from the community. We will prioritize this feature soon.

**Comment by lgabriel-fauth** [comment]
@muhsin-k Hi, is there any date for this feature to be implemented?

**Comment by roberto0arruda** [comment]
Hi folks! 👋

I’ve confirmed that it's possible to create new conversations through the API (via the `/api/v1/accounts/:account_id/conversations` endpoint), and I’d like to help implement this feature in the mobile app.

Before diving in, I’d really appreciate your insights on a few things:

1. Are there any limitations on which channels (e.g., WhatsApp, email) can be used to initiate a conversation through the API?

2. Should the mobile feature support starting a conversation only with existing contacts, or also allow messages to new recipients (e.g., phone numbers not yet in the system)?

3. Is there any suggested UI/UX for how the “New Conversation” feature should be integrated into the current mobile app flow?

Thanks in advance for your guidance! I'd be happy to contribute a PR once I better understand the intended scope and design.

Tags: question, help wanted, good first issue

**Comment by lgabriel-fauth** [comment]
@roberto0arruda i also want to be a tester too

**Comment by prjcarr** [comment]
@roberto0arruda are we close to a release for this feature, or is it still beta?

**Comment by roberto0arruda** [comment]
> [@roberto0arruda](https://github.com/roberto0arruda) are we close to a release for this feature, or is it still beta?

@prjcarr 
This feature has already been implemented in my published Android version.
[Take my app](https://play.google.com/store/apps/details?id=online.ardigital.connect)

**Comment by lksventura** [comment]
> > [@roberto0arruda](https://github.com/roberto0arruda) are we close to a release for this feature, or is it still beta?
> 
> [@prjcarr](https://github.com/prjcarr) This feature has already been implemented in my published Android version. [Take my app](https://play.google.com/store/apps/details?id=online.ardigital.connect)

Is there any possibility of you making a pull request with this new feature or making your app's repository available?
